using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,455 @@
+
+### Execute Request
+
+Look further down the API Playground page, in the "Response" area, and click on the button that says "Execute Request." You might get a "waiting" spinner, and then a box will appear below the button with the data that REDCap returned from your request.
+
+You requested your data to be in a .csv, so you should get some data that's "comma separated" – a bunch of fields that are separated by commas, with each new line of data being separated by a line break.
+
+<div class = "warning">
+
+In the "Response" box, data is presented in plain text, not in a table, so it might look confusing or overwhelming. If you want to, you can copy that plain text by clicking on the contents of the box and then using your browser's "Select All" function, followed by "Copy." Then paste it into a text editor like Atom or Notepad, saving it with the .csv extension. That will allow you to then open it in Excel to see if the .csv is what you intended. Below, here's the "before" (comma separated values in plain text) and the "after" (text saved as a .csv and then opened in Excel) of the cervical cancer data:
+
+![Data shown as text, separated by commas.](media/execute_request.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+![Data shown in an Excel spreadsheet.](media/excel_view.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+</div>
+
+Below the Response box with the comma separated values data, you'll see an HTTP status. You want that to be 200, which means no errors occurred.
+
+This is an example of trying out and learning about the API without having to write code. You can experiment with various methods and learn easily without having to go through the frustration of writing a lot of code to find out how to do things.
+
+In some API methods (like "Export Records"), you'll have a number of drop down menus you can choose from to tailor your request, including which "forms" and which "fields" you want to retrieve. You can make multiple choices in menus like these by holding down control while clicking (Windows) or command while clicking (Mac).  Helpfully, however, if you want **every** item listed (say, you want every form and every field), you don't click any selection at all, and REDCap assumes you mean everything that appears in the drop-down list.
+
+<div class = "important">
+Do take a look and make sure your "Response" box contains comma separated values and shows a good return code (200).  If not, something has gone wrong -- either you don't have the correct rights, which means you need to take another look at your rights in the User Rights section of REDCap, or you've issued an API call that is incorrect in some way (for instance, maybe you requested a .json format instead of .csv).
```

Comment:
```suggestion
Do take a look and make sure your Response box contains comma separated values and shows a good return code (200).  If not, something has gone wrong -- either you don't have the correct rights, which means you need to take another look at your rights in the User Rights section of REDCap, or you've issued an API call that is incorrect in some way (for instance, maybe you requested a .json format instead of .csv).
```